packages/core/testing/src/test_bed.ts

```diff
@@ -205,7 +205,7 @@ export class TestBedImpl implements TestBed {
    * Defer block behavior option that specifies whether defer blocks will be triggered manually
    * or set to play through.
    */
-  private _instanceDeferBlockBehavior = DeferBlockBehavior.Manual;
+  private _instanceDeferBlockBehavior = DeferBlockBehavior.Playthrough;
```

Comment:
I think it might be helpful to extract this to a separate const for easier refactoring in the future:

```suggestion
  private _instanceDeferBlockBehavior = DEFER_BLOCK_DEFAULT_BEHAVIOR;
```

Reply:
done